src/ucp/api/ucp.h

```diff
@@ -1452,6 +1470,23 @@ ucs_status_ptr_t ucp_disconnect_nb(ucp_ep_h ep);
 
 /**
  * @ingroup UCP_ENDPOINT
+ *
+ * @brief Blocking @ref ucp_ep_h "endpoint" closure.
```

Comment:
we should make it non-blocking, since it should serve as a replacement for ucp_ep_disconnect_nb()

Reply:
I sent email to the group with another option. Lets gather feedback when i'll update the PR.